tests/file_uploads/tests.py

```diff
@@ -435,6 +435,16 @@ def test_broken_custom_upload_handler(self):
             with self.assertRaisesMessage(AttributeError, msg):
                 self.client.post('/quota/broken/', {'f': file})
 
+    def test_upload_stopped(self):
+        with tempfile.NamedTemporaryFile() as file:
+            file.write(b'a' * (2**21))
+            file.seek(0)
+            path = file.name
+            self.assertTrue(os.path.isfile(path))
```

Comment:
Is there a reason to use `isfile()` instead of `exists()`.

Reply:
No real reason, don't really see the difference between using either in this case.